app/views/sessions/show.html.erb

```diff
@@ -32,16 +34,22 @@
         </p>
 
         <h2 class="nhsuk-heading-s nhsuk-u-margin-bottom-1">
-          <%= link_to "Triage health questions",
-            triage_session_path(@session) %>
+          <%= link_to(
+            "Triage health questions",
+            # TODO: Can we get rid of the section param here?
```

Comment:
Worth checking this in like a timeboxed ~5 minutes but I'd remove these TODOs before merging even if we don't. It's really not that bad having to specify it just in this instance.